packages/pretty-format/src/__tests__/DOMElement.test.ts

```diff
@@ -583,3 +583,50 @@ Testing.`;
     );
   });
 });
+
+describe('DOMElement Plugin with a null printer', () => {
+  beforeAll(() => {
+    setPrettyPrint([
+      {
+        serialize() {
+          return '';
+        },
+        test(value) {
+          return (
+            DOMElement.test(value) &&
+            (value.tagName === 'SCRIPT' || value.nodeType === 8)
+          );
+        },
+      },
+      DOMElement,
+    ]);
+  });
+
+  afterAll(() => {
+    setPrettyPrint([DOMElement]);
+  });
+
+  it('filters children', () => {
+    const div = document.createElement('div');
+
```

Comment:
If possible I think it would be easier to compare this to the `expected` if this was being constructed via
```
div.innerHTML = `...`
```